### DISCUSSION TITLE
[seek help] How to make use of Python to set location for a wifi connected iPhone?

### DISCUSSION THREAD

**Discussion by terryyuen** [doronz88/pymobiledevice3#740] [General]
I am beginner of Python. Would anyone can show me how to make use pymobiledevice3 to spoof the location for a wifi connected iPhone? Appreciate for your help.

**Comment by acg7878** [comment]
i want to know too

**Reply by doronz88** [reply]
Read the project's README:

```shell
# Simulate a `lat long` location (iOS < 17.0)
pymobiledevice3 developer simulate-location set -- lat long

# Simulate a `lat long` location (iOS >= 17.0)
pymobiledevice3 developer dvt simulate-location set -- lat long
```

**Reply by acg7878** [reply]
it's my pleasure to get your reply. And i want to know whether the tool could alter iphone's location by WIFI. I browser some information and find that i should use a tool named usbmux2?

**Reply by acg7878** [reply]
my english is not good. another question is that : could i use the pymobiledevice at armbian?   i want to use a phone to alter my iphone by WIFI

**Reply by doronz88** [reply]
Connecting to the device over wifi has different solutions for each platform you are using on for which iOS version you are targeting
